### DISCUSSION TITLE
16.1.2

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#859] [General]
## Releases Notes for 16.1.2
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v16.1.2/draw.io-16.1.2-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v16.1.2/draw.io-16.1.2-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v16.1.2/draw.io-universal-16.1.2.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v16.1.2/drawio-amd64-16.1.2.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v16.1.2/drawio-x86_64-16.1.2.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v16.1.2/drawio-x86_64-16.1.2.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Changes to move CPU intensive operations out of rendering process to fix stability issues in #853 
- Fixes #852 , #728 , #803 ,#842
- Updates to [draw.io core 16.1.2](https://github.com/jgraph/drawio/blob/v16.1.2/ChangeLog). All changes from 16.0.3 to 16.1.2, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v16.1.2'>16.1.2</a>.</em>

**Comment by ssravjee** [comment]
Great work guys. 
I am wondering when can we have the plantuml plugin  (Arrange /Insert /Advanced/Plantuml) that's available on the online version https://app.diagrams.net/ in his repo by?

**Reply by davidjgraph** [reply]
It's not possible.

**Reply by ssravjee** [reply]
I am sure there is a good reason why it's not possible. 

I was wondering what is the exact reason it cannot be achieved?
